Makefile

```diff
@@ -71,3 +71,7 @@ check-code:
 	yarn tsc-check
 	yarn eslint-check
 	yarn build
+
+debug:
```

Comment:
I don't think we need to add a command to the Makefile.
Can just run the script via yarn directly as we've been doing so far.

Reply:
Yup I need to remove the Makefile entry and hardcore `active: true` line. Just had those in for easy testing locally.